bluesky/callbacks/best_effort.py

```diff
@@ -432,7 +432,7 @@ def clear(self):
 
 
 class PeakResults:
-    ATTRS = ('com', 'cen', 'max', 'min', 'fwhm', 'nlls')
+    ATTRS = ('com', 'cen', 'max', 'min', 'fwhm', 'nlls', 'crossings')
```

Comment:
@danielballan, I think this part was missed in #888, so I added the `crossings` attribute here as well. This caused a problem with `test_bec.py` in Travis (also reproduced in my dev env). I solved it by changing the type of the `crossings` from `ndarray` to `list` (see [below](https://github.com/NSLS-II/bluesky/pull/1002/files#diff-177dabe93bbee02f265f7408541d9964R291)).